manifests/plugin/mcelog.pp

```diff
@@ -0,0 +1,17 @@
+# https://collectd.org/documentation/manpages/collectd.conf.5.shtml#plugin_mcelog
+class collectd::plugin::mcelog (
+  Enum['present', 'absent'] $ensure                       = 'present',
+  # Log file option and memory option are mutualy exclusive.
+  Optional[String]          $mceloglogfile                = '/var/log/mcelog',
+  Optional[Boolean]         $memory                       = undef,
+  Optional[String]          $mcelogclientsocket           = undef,
+  Optional[Boolean]         $persistentnotification       = undef,
+) {
+
+  include collectd
+
+  collectd::plugin { 'mcelog':
+    ensure  => $ensure,
+    content => template('collectd/plugin/mcelog.conf.erb'),
```

Comment:
where is this file?

Reply:
templates/plugin/mcelog.conf.erb

Its the same way intel_pmu is done. Am I incorrect in assuming that the correct path?